packages/driver/src/multi-domain/cypress.ts

```diff
@@ -69,7 +69,24 @@ const setup = () => {
 
   cy.onBeforeAppWindowLoad = onBeforeAppWindowLoad(Cypress, cy)
 
-  specBridgeCommunicator.initialize(window)
+  // outlaw the use of `route` and `server` within the multi-domain context and Cypress.Server.* configurations
+  // @ts-ignore
+  cy.route = () => $errUtils.throwErrByPath('switchToDomain.route.forbidden')
+  // @ts-ignore
+  cy.server = () => $errUtils.throwErrByPath('switchToDomain.server.forbidden')
+  Cypress.Server = new Proxy(Cypress.Server, {
+    get: () => $errUtils.throwErrByPath('switchToDomain.Server.forbidden'),
```

Comment:
Do we possibly need an `apply` handler here? It should be unlikely since these are all namespaced, but wanted to ask!

Reply:
I don't think you need an apply here because it's not valid to call the server as a function.
Calling `Cypress.Server()` would throw `Cypress.Server is not a function`.